packages/labs/react/src/create-component.ts

```diff
@@ -127,25 +145,32 @@ type EventProps<R extends Events> = {
  * messages. Default value is inferred from the name of custom element class
  * registered via `customElements.define`.
  */
-export const createComponent = <I extends HTMLElement, E extends Events>(
+export const createComponent = <
+  I extends HTMLElement,
+  E extends Events,
+  C extends Record<string, string>
+>(
   React: typeof ReactModule,
   tagName: string,
   elementClass: Constructor<I>,
   events?: E,
-  displayName?: string
+  displayName?: string,
+  children?: C
```

Comment:
I think it makes more sense to call this argument `slots`.

Reply:
Sounds good!